airflow/executors/kubernetes_executor.py

```diff
@@ -71,6 +71,8 @@ def __init__(self):  # pylint: disable=too-many-statements
                                                'env_from_configmap_ref')
         self.env_from_secret_ref = conf.get(self.kubernetes_section,
                                             'env_from_secret_ref')
+        self.delete_options_kwargs = conf.get(self.kubernetes_section,
```

Comment:
This is a string isn't it, but the doc says kwargs. Are we missing a `json.loads` here?

Reply:
Good catch, @ashb! Pushing up a quick fix now.